CITATION.cff

```diff
@@ -0,0 +1,396 @@
+  - given-names: Darío
+    family-names: Hereñú
+    email: magallania@gmail.com
+  - given-names: Stefano
+    family-names: Iacus
+    email: smiacus@gmail.com
+  - given-names: Alex
+    family-names: Ivanov
+    email: alex@calmforce.com
+  - given-names: Thomas
+    family-names: J. Leeper
+    email: thosjleeper@gmail.com
+  - given-names: Jamie
+    family-names: Jamison
+    email: jamison@library.ucla.edu
+  - given-names: Sebastian
+    family-names: Karcher
+    email: karcher@u.northwestern.edu
+  - given-names: Stefan
+    family-names: Kasberger
+    email: mail@stefankasberger.at
+  - given-names: Dominic
+    family-names: Kempf
+    email: dokempf@users.noreply.github.com
+  - given-names: Gary
+    family-names: King
+    email: king@harvard.edu
+  - given-names: Peter
+    family-names: Kiraly
+    email: peter.kiraly@gwdg.de
+  - given-names: Péter
+    family-names: Király
+    email: pkiraly@gwdg.de
+  - given-names: Tom
+    family-names: Koch
+    email: tomck@users.noreply.github.com
+  - given-names: Kacper
+    family-names: Kowalik (Xarthisius)
+    email: xarthisius.kk@gmail.com
+  - given-names: Stephen
+    family-names: Kraffmiller
+    email: skraffmiller@hmdc.harvard.edu
+  - given-names: Heinz
+    family-names: Kramski
+    email: kramski@web.de
+  - given-names: Karol
+    family-names: Kulik
+    email: kkulik@icm.edu.pl
+  - given-names: Eryk
+    family-names: Kulikowski
+    email: eryk.kulikowski@kuleuven.be
+  - given-names: Shiro
+    family-names: Kuriwaki
+    email: shirokuriwaki@gmail.com
+  - given-names: Sherry
+    family-names: Lake
+    email: shlake@virginia.edu
+  - given-names: Jonathan
+    family-names: Leitschuh
+    email: Jonathan.Leitschuh@gmail.com
+  - given-names: Maylein,
+    family-names: Leonhard
+    email: rt126@uni-heidelberg.de
+  - given-names: Victoria
+    family-names: Lubitch
+    email: victoria.lubitch@utoronto.ca
+  - given-names: Jing
+    family-names: Ma
+    email: ma.jing@outlook.com
+  - given-names: Benjamin
+    family-names: Martinez
+    email: benjamin.martiinez@gmail.com
+  - given-names: Esteban
+    family-names: Martinez
+    email: esteban.martinez@csuc.cat
+  - given-names: Leonhard
+    family-names: Maylein
+    email: Maylein@ub.uni-heidelberg.de
+  - given-names: Carlos
+    family-names: Mcgregor
+    email: c.mcgregormuro@mail.utoronto.ca
+  - given-names: Dan
+    family-names: Mcpherson
+    email: dmcphers@redhat.com
+  - given-names: Pete
+    family-names: Meyer
+    email: pameyer@crystal.harvard.edu
+  - given-names: Tommy
+    family-names: Monson
+    email: tmonson@redhat.com
+  - given-names: Pietro
+    family-names: Monticone
+    email: 38562595+pitmonticone@users.noreply.github.com
+  - given-names: Usman
+    family-names: Muchlish
+    email: u.muchlish@gmail.com
+  - given-names: Derek
+    family-names: Murphy
+    email: dlmurphy@g.harvard.edu
+  - given-names: Jim
+    family-names: Myers
+    email: myersjd@umich.edu
```

Comment:
FWIW: The method used to create this appears to have picked up my old/non-functional email and not included my more recent qqmyers/qqmyers@hotmail.com information. While that could be a quick edit here, does it mean there are others potentially in this situation? (Was qqmyers missed because there's no first/last name to parse?).

Reply:
Precisely the reason why the HERMES Git harvester supports a mailmap :wink: